core/app/services/spree/products/prepare_nested_attributes.rb

```
@@ -136,8 +136,15 @@ def update_option_value_variants(option_value_params, existing_variant)
             o.position = opt[:position]
             o.save!
           end
-          option_value = option_type.option_values.where(name: opt[:value].parameterize).first_or_initialize do |o|
-            o.name = o.presentation = opt[:value]
+
+          option_value_identificator = if opt[:option_value_id].present?
+            {id: opt[:option_value_id]}
```

RuboCop (1.75.5)

[convention] Use 2 (not -27) spaces for indentation.

(Layout/IndentationWidth)